content/blog/2017-12-07-introducing-the-react-rfc-process.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Introducing the React RFC Process"
+author: [acdlite]
+---
+
+We're adopting an RFC ("request for comments") process for contributing ideas to React. 
+
+Inspired by [Yarn](https://github.com/yarnpkg/rfcs), [Ember](https://github.com/emberjs/rfcs), and [Rust](https://github.com/rust-lang/rfcs), the goal is to allow React core team members and community members to collaborate on the design of new features. It's also intended to provide a clear path for ideas to enter the project:
+
+- Create an RFC document detailing your proposal.
+- Submit a PR to the [RFC repository](https://github.com/reactjs/rfcs).
+- Incorporate feedback into the proposal.
+- After discussion, the RFC may be accepted for implementation by the core team.
```

Comment:
Can it be implemented by somebody *outside* the team too?

Reply:
I guess “by” placement is ambiguous here.

Reply:
Yeah, I'll rephrase. (Though it's true that anyone is free to submit a PR and implement it.)